windows.md

```diff
@@ -390,6 +390,49 @@ With those compatibility things out of the way, you're ready to start the system
     4. Open the start menu and search for "Ubuntu". Start it - it should ask you to create a user with a password. This will be your user to log in to your Ubuntu Linux Subsystem - note down the username and password somewhere secure to make sure you do not forget it.
     5. Open the start menu and search for "Docker Desktop". Start it and go to the Settings. Under the General tab, you will find an option called "Use WSL 2 based engine". Make sure this is checked.
 
+    If opening Docker Desktop from **Option A or B** shows an error, you may need to enable virtualization on your machine
+
+    The error might look like this:<br><br>
+    <img src="docker-desktop-opening-error.avif"><br><br>
+
+    Close PowerShell and open it again as administrator (like in step 2)
+
+    Check if your computer supports virtualization. Run this command:
+
+    ```bash
+    Get-ComputerInfo -Property HyperV*
+    ```
+
+    This should display the following output:
+
+    ```bash
+    HyperVisorPresent: False
+    HyperVRequirementDataExecutionPreventionAvailable : True
+    HyperVRequirementSecondLevelAddressTranslation    : True
+    HyperVRequirementVirtualizationFirmwareEnabled    : False
+    HyperVRequirementVMMonitorModeExtensions          : True
+    ```
+
+    Check if virtualization is enabled. Run this command:
+
+    ```bash
+    systeminfo | findstr Virtualization
+    ```
+
+    This should display the following output:
+
+    ```bash
+    Virtualization Enabled In Firmware: No
+    ```
```

Comment:
can this be combined with the previous command?

Reply:
I think this can be removed as the information it provides is also available on the first command output
https://github.com/upleveled/system-setup/pull/16/commits/63fe30ab3bddc4701ef0e54ac5d1981c9ee84d24